src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs

```diff
@@ -71,6 +73,22 @@ internal static class CsvAppFileIO {
             Interlocked.Exchange(ref _busy, 0);
         }
 
+        public static void Close() {
+            var folder = GetTempCsvFilesFolder();
+            if (Directory.Exists(folder)) {
+                foreach (var f in Directory.EnumerateFiles(folder)) {
+                    try {
+                        File.Delete(f);
+                    } catch(IOException) { } catch (UnauthorizedAccessException) { }
+                }
+            }
+        }
+
+        private static string GetTempCsvFilesFolder() {
+            var folder = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);
```

Comment:
I don't understand why, though. I understand why you'd want to put these files into Documents when user is doing "Export to CSV". But in this scenario, which is "Open in Excel", these are transient files. Why would the user care where they live? They're going to be gone as soon as he closes VS, anyway.

@mortaz, can you clarify?